### PR TITLE
test(review): make the quote-verdict switch fully covered (follow-up #216)

### PR DESCRIPTION
**Pull request by devops-thiago**
## What type of PR is this?

- [x] ✅ Test
- [x] 🔧 Refactor

## Description

The enum `switch` in `FindingQuoteValidator.validate()` had no `default`, so javac emits a **synthetic default** branch that no test can reach — JaCoCo reports the switch line as *partially* covered. #216 changed that line, so the partial now sits on `main` and would count against patch coverage on the next PR that modifies the area.

Fix: fold the keep verdicts (`FULL`, `NO_QUOTE`) into an explicit `default` placed last. It's reached by the existing FULL/NO_QUOTE tests, so there's no synthetic default and the line is fully covered. `default`-keep is also the safe handling for any future verdict (it surfaces the finding rather than silently dropping it). **No behavior change.**

## Related Issues

Follow-up to #216. N/A (no separate issue).

## How Has This Been Tested?

- [x] Unit tests

- JaCoCo now reports **no partial branches** in `FindingQuoteValidator`; the existing `FindingQuoteValidatorTest` already covers all four arms (FULL/NO_QUOTE via keep tests, PARTIAL, AMBIGUOUS, NONE).
- Full suite: **1221 passing**, spotbugs + spotless clean (JDK 25).

## Checklist

- [x] My code follows the project's coding standards
- [x] I have performed a self-review of my own code
- [x] I have added tests that prove my fix is effective (coverage verified via JaCoCo)
- [x] New and existing unit tests pass locally with my changes
- [ ] I have updated the documentation accordingly (no doc change needed)
- [x] My changes generate no new warnings or errors

### CODE REVIEW

File: src/main/java/dev/thiagogonzaga/thrillhousebot/review/FindingQuoteValidator.java [hunk 1]
```diff
@@ -69,20 +69,6 @@ public ReviewResponse validate(ReviewResponse response, String diff) {
     var changed = false;
     for (ReviewResponse.Finding finding : response.findings()) {
       switch (classifyQuote(finding, index)) {
-        case FULL, NO_QUOTE -> {
```
Comment (devops-thiago) [inline src/main/java/dev/thiagogonzaga/thrillhousebot/review/FindingQuoteValidator.java:72]: @thrillhousebot, what's the impact of removing these case in switch?
